Cargo.toml

```diff
@@ -16,11 +16,14 @@ include = ["src/**/*", "LICENSE.md"]
 
 [dependencies]
 anyhow = "1.0.44"
-async-std = {version  = "1.10.0", features = ["unstable", "attributes"] }
 combine = "4.6.1"
 termion = "1.5.6"
 unic-ucd = "0.9.0"
```

Comment:
Could this be changed to unic-ucd-category now that the parser only uses that crate?